### ISSUE TITLE
Scan spec item from test report

### ISSUE THREAD

**Issue by redcatbear** [itsallcode/openfasttrace#183]
As a quality engineer
I want to check whether the test covering a requirement were executed and succeeded
in order to make sure that tests not only exist, but are in fact used and come back green.

## Implementation Idea
Since #159 we are using JUnit5 as unit testing framework it should be possible to find or write a JUnit5 extension that lets us add the specification item ID to the test name so that it will appear in test reports.

OFT should then scan the test results and correlate them with the tests, allowing for even deeper coverage checks.

**Comment by redcatbear** [comment]
@kaklakariada, I think this would be quite an advantage over a lot of existing systems.

**Comment by kaklakariada** [comment]
Good idea. I propose to use a custom annotation on test methods. This will require an additional importer.

**Comment by redcatbear** [comment]
Exactly. We then also need a priority mechanism that makes sure that if there are two competing importers for one source, the that is assumed better is tried first.

If that fails to produce any items, we fall back to the next.
